RecoEgamma/EgammaIsolationAlgos/python/pfClusterIsolationRemapForPAT_cff.py

```diff
@@ -0,0 +1,30 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoEgamma.EgammaIsolationAlgos.pfClusterIsolationRemap_cfi import *
+
+electronHcalPFClusterIsolationProducerGSFixed = electronHcalPFClusterIsolationRemapper.clone(
```

Comment:
also, the point here is to not propagate objects from pfClusterIsolationRemap_cfi into the global space.
So
``` 
from RecoEgamma.EgammaIsolationAlgos.pfClusterIsolationRemap_cfi import electronHcalPFClusterIsolationRemapper as _electronHcalPFClusterIsolationRemapper ```